src/app/view/ChessBoardView.ts

```diff
@@ -19,3 +20,47 @@ export interface ChessBoardView {
   //showAvailableMoves
   //showSelectedPiece
 }
+*/
+
+export class Square {
+  constructor(private position_x: number, private position_y: number, private color: SquareColor) {}
+
+  setHtmlElement(): HTMLElement {
+    const position = this.mapSquarePosition();
+    const squareElement = document.createElement('div');
+    squareElement.id = position;
+    squareElement.classList.add(`square--${this.color}`);
+    return squareElement;
+  }
+
+  mapSquarePosition(): string {
+    const columns = 'abcdefgh'.split('').reverse();
+    const rows = '12345678'.split('');
```

Comment:
sprytne, ale mamy już to w pliku ze stałymi.
Choć może mylę.
Zaciągnij "develop'a"

Reply:
Tutaj akurat bym zostawił :) 
Bo tamte stałe są do modelu, a tutaj mamy widok. Co prawda może dałoby się wydzielić jakąś część wspólną - ale na razie tego nie róbmy. Duplikacja samego kodu nie zawsze jest zła - o wiele gorsze może być powiązanie widoku z logiką.

Reply:
Super, ze to napisałeś. Zapomniałem się, że jesteśmy teraz w VIEW.
W takim razie cofam to co powyżej.
Dzięki za uwagę. Kolejnej rzeczy się nauczyłem!

Reply:
Widziałam, ale świadomie tego nie wrzucałam :)